mongo/pom.xml

```diff
@@ -90,5 +90,21 @@
       <version>1.10.19</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-core</artifactId>
+      <version>2.8.8</version>
+    </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-databind</artifactId>
+      <version>2.8.8</version>
```

Comment:
need probably at least 2.8.11 for jeckson, and 2.8.11.1 for the jackson-databind, or newer (vulnerabilities are alerted)